python/src/scipp/html/style.css.template

```diff
@@ -3,6 +3,22 @@
 * https://github.com/jsignell/xarray/blob/1d960933ab252e0f79f7e050e6c9261d55568057/xarray/static/css/style.css
 */
 
+:root {
+  --sc-background-color0: var(--jp-layout-color0, #fff);
+  --sc-background-color1: var(--jp-layout-color1, #fcfcfc);
+  --sc-background-color2: var(--jp-layout-color2, #efefef);
+  --sc-inverse-background-color0: var(--jp-inverse-layout-color4, #111);
+  --sc-font-color0: var(--jp-content-font-color0, #000);
```

Comment:
Are these names `*-color0` defined by jupyter?
It would be clearer to have more explicit names. instead of number suffixes, such as `sc-background-color-lighter` or `sc-background-color-white`, but maybe these are the names you have to use?

Reply:
The `--jp-` properties are defined by jupyter.
'lighter', 'darker', or 'white' do not work because what is lighter in the light theme is darker in the dark theme.